modules/installation-gcp-permissions.adoc

```diff
@@ -32,6 +32,13 @@ account requires the following permissions. If you deploy your cluster into an e
 .Required roles for creating network resources during installation
 * DNS Administrator
 
+.Required roles for installing into a shared VPC
+* Compute Instance Admin (v1)
```

Comment:
Sorry, this role can be removed from the list too, because it's a subset of the role `roles/compute.admin`, thanks!